Dockerfile.ci

```diff
@@ -660,7 +660,7 @@ function install_airflow() {
 
         # Similarly we need _a_ file for task_sdk too
         mkdir -p ./task_sdk/src/airflow/sdk/
-        touch ./task_sdk/src/airflow/__init__.py
+        touch ./task_sdk/src/airflow/sdk/__init__.py
```

Comment:
This path was okay before, but since I added `exclude = ["src/airflow/__init__.py"]` in this PR this path now needs to change.